defaults/host-ents.go

```diff
@@ -0,0 +1,197 @@
+package defaults
+
+import (
+	"github.com/docker/libentitlement/entitlement"
+	"github.com/docker/libentitlement/secprofile"
+	"github.com/docker/libentitlement/types"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"reflect"
+)
+
+const (
+	hostDomain = "host"
+
+	hostProcessesDomain = hostDomain + ".processes"
+	hostDevicesDomain   = hostDomain + ".devices"
+)
+
+const (
+	// HostDevicesNoneEntFullID is the ID for the host.devices.none entitlement
+	HostDevicesNoneEntFullID = hostDevicesDomain + ".none"
+	// HostDevicesViewEntFullID is the ID for the host.devices.view entitlement
+	HostDevicesViewEntFullID = hostDevicesDomain + ".view"
+	// HostDevicesAdminEntFullID is the ID for the host.devices.admin entitlement
+	HostDevicesAdminEntFullID = hostDevicesDomain + ".admin"
+
+	// HostProcessesNoneEntFullID is the ID for the host.processes.none entitlement
+	HostProcessesNoneEntFullID = hostProcessesDomain + ".none"
+	// HostProcessesAdminEntFullID is the ID for the host.processes.admin entitlement
+	HostProcessesAdminEntFullID = hostProcessesDomain + ".admin"
+)
+
+var (
+	hostDevicesNoneEntitlement  = entitlement.NewVoidEntitlement(HostDevicesNoneEntFullID, hostDevicesNoneEntitlementEnforce)
+	hostDevicesViewEntitlement  = entitlement.NewVoidEntitlement(HostDevicesViewEntFullID, hostDevicesViewEntitlementEnforce)
+	hostDevicesAdminEntitlement = entitlement.NewVoidEntitlement(HostDevicesAdminEntFullID, hostDevicesAdminEntitlementEnforce)
+
+	hostProcessesNoneEntitlement  = entitlement.NewVoidEntitlement(HostProcessesNoneEntFullID, hostProcessesNoneEntitlementEnforce)
+	hostProcessesAdminEntitlement = entitlement.NewVoidEntitlement(HostProcessesAdminEntFullID, hostProcessesAdminEntitlementEnforce)
+)
+
+var (
+	allowedMounts = []specs.Mount{
```

Comment:
can this be a `const`?  Also, can we rename this to something more descriptive to represent host devices and sysfs?